### ISSUE TITLE
Dice score is not change 

### ISSUE THREAD

**Issue by JonyJiang123** [milesial/Pytorch-UNet#305]
Recently, I got the problem when trained model with own data.

![image](https://user-images.githubusercontent.com/75117639/137444954-86d3ac1b-cf50-4410-857f-eb5dc7254197.png)

Dice score is  unchanged and close to 0， but the same data is in another unet version(keras),  test result is good.

INFO: Starting training:
        Epochs:          50
        Batch size:      16
        Learning rate:   0.0001
        Training size:   232
        Validation size: 25
        Checkpoints:     True
        Device:          cuda
        Images scaling:  1.0
        Mixed Precision: False

**Comment by Huster-Hq** [comment]
You can try changing the optimizer

**Comment by nqthai309** [comment]
have you solved it yet? help me

**Comment by VladDok** [comment]
@biel-jony, try to reduce learning rate and check result.

**Comment by leosmaia21** [comment]
Hello, did you manage to solve it?
